aten/src/ATen/native/Convolution.cpp

```diff
@@ -504,6 +523,8 @@ at::Tensor convolution_overrideable(
   AT_ERROR("You are likely triggering this with tensor backend other than CPU/CUDA/MKLDNN, if this is intended, please use globalATenDispatch().registerOp to override this function ");
 }
 
+DEFINE_DISPATCH(convolution_depthwise3x3_winograd);
```

Comment:
Is this an OK place to add this?

Reply:
yes, but I'd probably move it towards the top of the file to make more discoverable.

also other names seems to end in `_stub` so maybe follow the same convention

Reply:
Thank you for your comments.  I addressed them and will upload a patch shortly.